docs/low-level-api.md

```diff
@@ -11,30 +11,24 @@ import { actionToPath, pathToAction } from 'redux-first-router'
 const { routesMap } = store.getState().location
 
 const path = actionToPath(action, routesMap, querySerializer)
-const action = pathToAction(path, routesMap, querySerializer, basename)
+const action = pathToAction(path, routesMap, querySerializer, basename, strict)
 ```
 
-The `querySerializer` argument is optional and the same as the one passed to `connectRoutes`.
+The `querySerializer`, `basename`, and `strict` arguments are optionals.
+
+The `querySerializer` argument is the same as the one passed to `connectRoutes`.
 It defaults to undefined.
 
-The `basename` argument to `pathToAction` is optional and works the same way as the one passed to routesMap.
-It defaults to the value passed the last time `connectRoutes` was called.
+The `basename` and `strict` arguments default to the value passed the last time `connectRoutes` was called.
+The `basename` argument works the same way as the one passed to `routesMap`.
 `actionToPath` does not apply any `basename` transformation.
+The `strict` option allows an optional trailing delimiter to match.
```

Comment:
What about "When `strict` is `true`, the presence or absence of a trailing slash is required to match the route path."?

Reply:
I couldn't have said it more concisely. Note: it as was a copy/paste from the description of `path-to-regexp`.